### ISSUE TITLE
Packages - search more item categories

### ISSUE THREAD

**Issue by Elderly-Emre** [DinoDevs/GladiatusCrazyAddon#383]
small grindstone + shred of leather
Grindstone + Protective gear

Can you add 2 more options to find them?

![image](https://user-images.githubusercontent.com/95325400/179570363-db768cd8-aed9-4487-9850-6f3ba443ebe6.png)

**Comment by mattemre** [comment]
I later learned that it belonged to gladiatus.
If not possible, this request can be removed

**Comment by FrutyX** [comment]
Yes, it is from Gladiatus, GCA is only adding the event category, but I remember correctly (I am not sure tho) even that is in the Gladiatus code, only it wasn't added by Gameforge.

I'll leave it to Thanos or Apo, if not possible, do close it.

**Comment by GreatApo** [comment]
It's exactly as fruty said. This will be possible only through searching with a search word

**Comment by mattemre** [comment]
then we can close this feature request

**Comment by GreatApo** [comment]
> then we can close this feature request

While it may not be that straight forward to add the feature, I see the usefulness of searching for specific items. Let's leave it open.

**Comment by FrutyX** [comment]
search word will not work with many different languages, unless you get all of them from the UI...

**Comment by FrutyX** [comment]
Hello!

This Issue has been selected to be merged into our Feature Requests & Enhancements overview Issue -  #530.

Please note that this does not change anything, the request is still taken in account, but it's moved for better and cleaner GitHub management.

Thank you for your feedback and ideas! We appreciate them :).
